### PR TITLE
remove identity picker from code

### PR DESCRIPTION
**Pull request by mohitbagra**

### CODE REVIEW

File: vss-extension.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "manifestVersion": 1.0,
     "id": "vsts-extensions-multivalue-control",
-    "version": "1.0.7",
+    "version": "1.0.6",
```
Comment (mmanela) [inline vss-extension.json]: Version change?

Comment (mohitbagra) [inline vss-extension.json]: on marketplace, the version is 1.0.5. So I just changed to the next version.

File: package.json [hunk 2]
```diff
@@ -26,13 +26,13 @@
...
     "tfx-cli": "^0.4.5",
-    "typescript": "^2.2.2",
+    "typescript": "2.3.4",
     "typings": "^2.1.0",
     "uglifyjs-webpack-plugin": "^0.4.2",
     "copy-webpack-plugin": "^4.0.1"
   },
   "dependencies": {
-    "vss-web-extension-sdk": "^2.114.0"
+    "vss-web-extension-sdk": "2.114.0"
```
Comment (mmanela) [inline package.json]: Why drop the carret

Comment (mohitbagra) [inline package.json]: Caret will download the latest compatible version and I have seen build issues with latest vss sdk and typescript. This is just to be safe

**Pull request merged**